complete/library/src/test/java/com/example/multimodule/service/MyServiceTest.java

```diff
@@ -18,7 +17,7 @@
 
     @Test
     public void contextLoads() {
-        assertThat(myService.message()).isNotNull();
+        assertThat(myService.message() != null, is(true));
```

Comment:
Ideally this would be reverted and we'd stick to AssertJ. If we move to Hamcrest this doesn't look idiomatic to me. It should be something like `assertThat(myService.message(), is(notNullValue()));`

Reply:
Looks like the one on the app was reverted to assertj but this one wasn't yet.

Reply:
I missed that one. Sorry. Thanks for catching it. Fixed now.